integration_tests/deltalake-sink/create_sink.sql

```diff
@@ -0,0 +1,20 @@
+create sink delta_lake_sink from source
+with (
+    connector = 'deltalake',
+    type = 'append-only',
+    location = 's3a://deltalake/delta',
+    s3.access.key = 'hummockadmin',
+    s3.secret.key = 'hummockadmin',
+    s3.endpoint = 'http://minio-0:9301'
+);
+
+-- Wait docker image version
```

Comment:
The java version can be deprecated soon after the rust version is well tested. The user interface should be the same even if we change the internal implementation. So we can ignore the case `deltalake_rust` in the demo.

Reply:
The commented script reserved for `deltalake_rust` in other files can also be removed.